kzz7yh-f63107/cod-ve09v2_kzz7yh-f63107.xml

```diff
@@ -110,8 +111,8 @@
       cu<lb ed="#V" n="56" break="no"/>piditas non superbia. 
    </p>
                <p xml:id="kzz7yh-f63107-d1e189">
-                  <lb ed="#V" n="57"/>Contra Eccl. i0 initium omnis peccati est superbia 
-      <lb ed="#V" n="58"/>Sed secundum quod dicitur Eccls. 25. A muliere 
+                  <lb ed="#V" n="57"/>Contra <ref>Eccl. 10</ref>initium omnis peccati est superbia 
```

Comment:
careful with your spacing here. We need to preserve a space after the ref and before the next word so that "10" and "initium" are recognized as separate words. 

```suggestion
                  <lb ed="#V" n="57"/>Contra <ref>Eccl. 10</ref> initium omnis peccati est superbia 
```